### PR TITLE
ADEN-6452 Introduces new AdEngine 3 API

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

**Comment by fraszczakszymon** [comment]
Is it necessary to keep bridge in ES6 here? Can we use built ES5 ad-engine/ad-products, `require` needed modules and get rid of build step of ad-products in app repository?

File: extensions/wikia/AdEngine/.babelrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      }]
+    ],
+    "plugins": [
+      "transform-class-properties",
+      "transform-object-rest-spread",
+      "transform-decorators-legacy",
+      "lodash"
+    ]
+  }
+
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/.babelrc]: Something is wrong with indents here ;)

Comment (leshniak) [reply extensions/wikia/AdEngine/.babelrc]: fixed

**Comment by leshniak** [comment]
We decided to consider bridge in ES5 in a follow up ticket, because it will change the scope of the current ticket. (discussed offline).

**Pull request merged**